src/constants/responseMessages.ts

```diff
@@ -0,0 +1,15 @@
+export const responseMessages = {
+    200: "OK. Everything worked as expected.",
+    201: "A resource was successfully created.",
+    204: " The request was handled successfully and the response contains no body content.",
+    304: "The resource was not modified. You can use the cached version.",
+    400: "Bad request.",
+    401: "Authentication failed.",
+    403: "The authenticated user is not allowed to access the specified API endpoint.",
+    404: "The requested resource does not exist.",
+    405: "Method not allowed. Please check the Allow header for the allowed HTTP methods.",
+    415: "Unsupported media type. The requested content type or version number is invalid.",
+    422: "Data validation failed.",
+    429: "Too many requests. The request was rejected due to rate limiting.",
+    500: "Internal server error.",
+} as {[key: number]: string}
```

Comment:
tento typ: `as {[key: number]: string}`  je podle mě úplně zbytečný

Reply:
AlertMessage komponenta s tym má problém, ked to neni otypované